src/components/accounts/accounts.js

```diff
@@ -0,0 +1,119 @@
+import template from "./accounts.html";
+import accountItemTemplate from "./account-item.html";
+import { ButtonMoreComponent } from "../button-more/button-more";
+import { AccountListService } from "../../services/account-service";
+import { AddAccountDialogComponent } from "../add-account-dialog/add-account-dialog";
+
+export class AccountsComponent {
+  constructor(mountPoint, props) {
+    this.mountPoint = mountPoint;
+    this.props = props;
+  }
+
+  querySelectors() {
+    this.accountsRoot = this.mountPoint.querySelector(".accounts");
+    this.moreBtnMountPoints = this.mountPoint.querySelectorAll(
+      ".accounts__more-button"
+    );
+    this.addAccountButton = this.mountPoint.querySelector(
+      ".accounts__add-account-dialog-activation"
+    );
+    this.addAccountMountPoint = document.querySelector(
+      ".accounts__add-account-dialog"
+    );
+    this.accountsList = this.mountPoint.querySelector(".accounts__list-items");
+  }
+
+  handleAddAccountConfirmed(account) {
+    AccountListService.add(account);
+    this.addAccountToHead(account);
+  }
+
+  handleAddAccountClicked() {
+    this.addAccountDialogComponent.showDialog();
+  }
+
+  initMoreBtns() {
+    this.moreBtnMountPoints = this.mountPoint.querySelectorAll(
```

Comment:
you do `this.moreBtnMountPoints = this.mountPoint.querySelectorAll` twice. here and on the row 15. is it "ok" or "typo"?